sqlmesh/core/plan/definition.py

```diff
@@ -158,15 +158,29 @@ def set_start(self, new_start: TimeLike) -> None:
         self._start = new_start
         self.__missing_intervals = None
 
+    @property
+    def _default_end(self) -> TimeLike:
```

Comment:
what's the purpose of this? can you delete this and override_end?

Reply:
We refer to the default end in two places (that being `now()`) so I wanted to make sure they were consistent